orchagent/isolationgrouporch.h

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright 2019 Broadcom.  The term "Broadcom" refers to Broadcom Inc.
+ * and/or its subsidiaries.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef __ISOLATIONGROUPORCH_H__
+#define __ISOLATIONGROUPORCH_H__
+
+#include "orch.h"
+#include "port.h"
+#include "observer.h"
+
+#define ISOLATION_GRP_DESCRIPTION       "DESCRIPTION"
+#define ISOLATION_GRP_TYPE              "TYPE"
+#define ISOLATION_GRP_PORTS             "PORTS"
+#define ISOLATION_GRP_MEMBERS           "MEMBERS"
+#define ISOLATION_GRP_TYPE_PORT         "port"
+#define ISOLATION_GRP_TYPE_BRIDGE_PORT  "bridge-port"
+
+typedef enum IsolationGroupType
```

Comment:
The following prefixes are used in this file
```
IsolationGroup (ISOLATION_GROUP_)
(IsoGrp) ISO_GRP_
ISOLATION_GRP_
```
I think it would be better to follow single named scheme.

Reply:
Thanks for the comment, yes at some places shorthand names are used. we will convert all the code in future check-ins.